docs/smtchecker.rst

```diff
@@ -34,16 +33,27 @@ The other verification targets that the SMTChecker checks at compile time are:
 - Out of bounds index access.
 - Insufficient funds for a transfer.
 
-All the targets above are automatically checked by default if all engines are
-enabled, except underflow and overflow for Solidity >=0.8.7.
+By default, for Solidity >=0.8.20 only assertions are checked.  For Solidity
+>=0.8.7 all the targets above are checked by default except for underflow and
+overflow.  Underflow and overflow are also in the default set for Solidity
+<0.8.7.  The above only applies for the default targets which can be
```

Comment:
```suggestion
<0.8.7. The above only applies for the default targets, which can be
```
Now I'm kinda curious. Are you using double whitespaces on purpose?

Reply:
No, I just use vim's automated `gq` which aligns the lines and ends up adding extra whitespaces sometimes